snapcraft.yaml

```diff
@@ -18,8 +18,7 @@ summary: Mozilla Thunderbird email application
 description: Thunderbird is a free email application that’s easy to set up and customize - and it’s loaded with great features!
 confinement: strict
 grade: stable
-base: core22
-compression: lzo
```

Comment:
Is changing compression from lzo to xz really intended? How does it relate to switching to core24.

Reply:
You are right it's not directly part of porting to core24, we had a discussion earlier this year about why libreoffice was lzo compressed despite not having that line and the (wrong?) conclusion was that the snapd default changed
(see the discussion on https://matrix.to/#/!GGqzbFAUQprdPgYYCM:ubuntu.com/$MX64gVkzlXTOAzCbnX19KxpJVDzXQEKwrmQ2qGzAe5I?via=ubuntu.com&via=matrix.org&via=envs.net)

but
```
$ sudo file /var/lib/snapd/snaps/thunderbird_492.snap 
/var/lib/snapd/snaps/thunderbird_492.snap: Squashfs filesystem, little endian, version 4.0, xz compressed, 130287863 bytes, 890 inodes, blocksize: 131072 bytes, created: Tue Jul  2 18:41:00 2024
```

so the beta/core24 build is indeed now .xz compressed... I've reverted that line deletion and force pushed, thanks for catching!